doc/api/crypto.md

```diff
@@ -1046,6 +1046,76 @@ encoding of `'utf8'` is enforced. If `data` is a [`Buffer`][], `TypedArray`, or
 
 This can be called many times with new data as it is streamed.
 
+## Class: KeyObject
```

Comment:
Would we want to add this as experimental first?

Reply:
That might be a good idea. Would that require us to put it behind a flag or could it be "documentation-only" experimental?

Reply:
The API is straightforward enough, I'd be ok with it not being experimental.

Reply:
If it did land as experimental, it would not need a flag as the behavior is entirely opt in. A runtime warning would be sufficient. That said, we did not land the new key generation APIs as experimental and I don't think we should really have to land this one that way either.

Reply:
After talking to @sam-github, I agree that this does not need to be experimental.